### ISSUE TITLE
growlight isn't drawing until we move over a device

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/growlight#75]
When I start the notcurses growlight (1.2.0, notcurses 1.1.1), I get no drawing until I actively do things:

![2020-01-28-223128_722x1417_scrot](https://user-images.githubusercontent.com/143473/73325837-03e4cd80-421e-11ea-8ae7-14be5de15b3c.png)

**Comment by dankamongmen** [comment]
I think this is a notcurses problem, as I haven't made any changes to growlight recently that would cause this.

**Comment by dankamongmen** [comment]
Actually, everything *is* getting drawn, but then when we're done, the screen is getting erased. We see everything again when we exit the program. Odd.

**Comment by dankamongmen** [comment]
If I tag up `ncplane_create()` with diagnostics, we see:

```c
 [schwarzgerat](0) $ cat e
Term: 80x74 xterm-256color (xterm with 256 colors)
NEW ONE 74/80 + 0/0
NEW ONE 5/19 + 68/0
NEW ONE 73/80 + 0/0
NEW ONE 73/80 + 0/0
NEW ONE 2/80 + 13/0
NEW ONE 2/80 + 19/0
NEW ONE 2/80 + 25/0
NEW ONE 2/80 + 33/0
NEW ONE 2/80 + 42/0
NEW ONE 2/80 + 48/0
NEW ONE 5/20 + 68/0
```

that last one looks suspicious

**Comment by dankamongmen** [comment]
that's rows/cols/yoff/xoff btw

**Comment by dankamongmen** [comment]
If I run against notcurses-1.1.0, this doesn't happen. It does if I run against notcurses-1.1.1.

**Comment by dankamongmen** [comment]
The issue hinges on when we emit `smcup`.

**Comment by dankamongmen** [comment]
This is fixed by requiring notcurses 1.1.2+.
